vars.go

```diff
@@ -8,274 +8,470 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+func newMetric(subsys string, name string, descr string) *prometheus.Desc {
```

Comment:
Yep, newDesc is a more descriptive name.

Ah, didn't know that about function definitions. I'll change it.